tests/p2p/test_state_sync.py

```diff
@@ -22,22 +23,23 @@ def make_random_state(n):
         code = b'not-real-code'
         account_db.set_code(addr, code)
         contents[addr] = (balance, nonce, storage, code)
-    return account_db, contents
+    account_db.persist()
+    return raw_db, account_db.state_root, contents
 
 
 def test_state_sync():
```

Comment:
This seems like a good candidate for using `hypothesis.strategies.random_module`

https://hypothesis.readthedocs.io/en/latest/data.html#hypothesis.strategies.random_module

You'll also need to convert the `make_random_state` function to use the `random` module for randomness generation.

Reply:
I think using hypothesis won't buy us anything in those tests: https://github.com/ethereum/py-evm/pull/279#issuecomment-358521463

Reply:
Hrm, I think you are correct, though I also don't think it will cost us anything, and in the event that there is some *strange* randomness based test failure, it will be really nice to be able to reproduce it, which now, we wouldn't be able to do.